ci/jenkinsfiles/release.groovy

```diff
@@ -0,0 +1,332 @@
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+* Contributors:
+*     Abdoul BA <aba@nuxeo.com>
+*/
+
+def appName = 'nuxeo-coldstorage'
+def pipelineLib
+def repositoryUrl = 'https://github.com/nuxeo/nuxeo-coldstorage/'
+
+properties([
+  [
+    $class: 'BuildDiscarderProperty',
+    strategy: [
+      $class: 'LogRotator',
+      daysToKeepStr: '15', numToKeepStr: '10',
+      artifactNumToKeepStr: '5'
+    ]
+  ],
+  [
+    $class: 'GithubProjectProperty', projectUrlStr: repositoryUrl
+  ],
+  disableConcurrentBuilds(),
+])
+
+String currentVersion() {
+  return readMavenPom().getVersion()
+}
+
+String getReleaseVersion(String version) {
+  return version.replace('-SNAPSHOT', '')
+}
+
+pipeline {
+  agent {
+    label 'builder-maven-nuxeo-11'
+  }
+  parameters {
+    string(name: 'BUILD_VERSION', description: 'Version to be promoted')
+    string(name: 'reference', description: 'Reference branch to be bumped after releasing')
+    booleanParam(
+      name: 'dryRun', defaultValue: true,
+      description: 'if true all steps will be run without publishing the artifact'
+    )
+  }
+  environment {
+    APP_NAME = "${appName}"
+    BACKEND_FOLDER = "${WORKSPACE}/nuxeo-coldstorage"
+    BRANCH_NAME = GIT_BRANCH.replace('origin/', '')
+    BRANCH_LC = "${BRANCH_NAME.toLowerCase().replace('.', '-')}"
+    CONNECT_PROD_URL = 'https://connect.nuxeo.com/nuxeo'
+    CHART_DIR = 'ci/helm/preview'
+    CURRENT_VERSION = currentVersion()
+    ENABLE_GITHUB_STATUS = 'false'
+    FRONTEND_FOLDER = "${WORKSPACE}/nuxeo-coldstorage-web"
+    JENKINS_HOME = '/root'
+    MAVEN_DEBUG = '-e'
+    MAVEN_OPTS = "${MAVEN_OPTS} -Xms512m -Xmx3072m"
+    ORG = 'nuxeo'
+    VERSION = getReleaseVersion(CURRENT_VERSION)
+  }
+  stages {
+    stage('Check parameters') {
+      steps {
+        script {
+          if (!params.BUILD_VERSION || params.BUILD_VERSION == '') {
+            currentBuild.result = 'ABORTED'
+            currentBuild.description = 'Missing required version parameter, aborting the build.'
+            error(currentBuild.description)
+          }
+
+          if (!params.reference || params.reference == '') {
+            currentBuild.result = 'ABORTED'
+            currentBuild.description = 'Missing required reference parameter, aborting the build.'
+            error(currentBuild.description)
+          } else {
+            echo '''
+              ----------------------------------------
+              Update Reference Branch
+              ----------------------------------------
+            '''
+            env.REFERENCE_BRANCH = params.reference
+          }
+
+          if (params.dryRun) {
+            env.DRY_RUN_RELEASE == 'true'
```

Comment:
params.dryRun will satisfy the check only when it's `true` right? I mean if we get anything else it will not be evaluated as true